### PR TITLE
Fixed deprecation warnings in I18nRouter

### PR DESCRIPTION
**Pull request by XWB**

### CODE REVIEW

File: Router/I18nRouter.php [hunk 1]
```diff
@@ -115,9 +109,12 @@ public function generate($name, $parameters = array(), $absolute = false)
             $locale = $this->defaultLocale;
         }
 
+        $absolute = self::ABSOLUTE_URL === $referenceType;
```
Comment (stof) [inline Router/I18nRouter.php]: this is not true. The scheme relative type also needs the right host

Comment (XWB) [inline Router/I18nRouter.php]: Not sure what you mean?

**Comment by acasademont** [comment]
Hi @XWB thaks for the patch! Some of the issues have already been fixed with some of the old PR's. Could you rebase from master and update this one and #178?

Thanks!

**Comment by acasademont** [comment]
Fixed the problems in the tests in another commit, thanks!

**Pull request closed without merging**